packages/wallet/src/redux/channelState/challenging/state.ts

```diff
@@ -24,10 +30,11 @@ export function challengeTransactionFailed<T extends ChannelOpen>(
     type: CHALLENGE_TRANSACTION_FAILED,
     stage: CHALLENGING,
     ...channelOpen(params),
+    channelType: 'Application',
```

Comment:
This should not be a magic constant, but should be defined
`const APPLICATION = 'APPLICATION'`
```
...
channelType: APPLICATION,
```

Or maybe it should even be an enum

Reply:
This has been obsoleted now but `channelType` is a string enum.

Reply:
ahhh, :+1: